### DISCUSSION TITLE
tutorials/making-games-shooter

### DISCUSSION THREAD

**Discussion by giscus** [ct-js/docs.ctjs.rocks#105] [Pages' discussions]
# tutorials/making-games-shooter

Let's make a small space shooting game with asteroids, lasers and hostile gunships! This tutorial will teach you how to import assets, handle user input, move things around and ...

http://localhost:8080/tutorials/making-games-shooter.html

**Comment by CosmoMyzrailGorynych** [comment]
Comments on docs.ctjs.rocks? Impossible!
